packages/docs/connections/Knex.yaml

```diff
@@ -0,0 +1,159 @@
+
+            The Knex connection can be used to execute raw SQL queries using the `KnexRaw` requests, or the Knex query builder can be used with the `KnexBuilder` request.
+
+            ## Connections
+
+            Connection types:
+              - Knex
+
+            ### Knex
+
+            #### Properties
+            - `client: enum`: __Required__ - The database client to use. One of:
+              - `mssql`
+              - `mysql`
+              - `mysql2`
+              - `oracledb`
+              - `postgres`
+              - `pg` (alias of `postgres`)
+              - `postgresql` (alias of `postgres`)
+              - `redshift`
+              - `sqlite3`
+              - `sqlite` (alias of `sqlite3`)
+            - `connection: object | string`: __Required__ - Connection string or object to pass to the database client. See the specific client documentation for more details.
+            - `searchPath: string`:  Set PostgreSQL search path.
+            - `version: string`:  Set database version.
+
+            #### Examples
+
+            MySQL:
+            ```yaml
+            connections:
+              - id: mysql
+                type: Knex
+                properties:
+                  client: postgres
+                  connection:
+                    host: '127.0.0.1'
+                    user : my_database_user
+                    password : my_database_password
+                    database : my_database
+            ```
+
+            PostgreSQL with connection string:
+            ```yaml
+            connections:
+              - id: postgres
+                type: Knex
+                properties:
+                  client: postgres
+                  connection:
+                    _secret: PG_CONNECTION_STRING
+            ```
+
+            ## Requests
+
+            Request types:
+              - KnexBuilder
+              - KnexRaw
+
+            ### KnexBuilder
+
+            #### Properties
+            - `query: object[]`: __Required__ - SQL query builder array. An array of objects, with a single key which is the name of the knex builder function. The value should be an array of arguments to pass to the builder function.
+            - `tableName: string | object`: The name of the table to query from.
+
+            #### Examples
+
+            Build a query:
+            ```yaml
+            id: knexBuilder
+            type: KnexBuilder
+            connectionId: knex
+            properties:
+              query:
+                - select:
+                    - '*'
+                - from:
+                    - users
+                - where:
+                    - name
+                    - _state: name
+            ```
+
+            ### KnexRaw
+
+            #### Properties
+            - `query: string`: __Required__ - SQL query string.
```

Comment:
Not seeing tableName in the connection, so I guess all these should have a tableName parameter? Also in the examples.